test/e2e/k8s_util.go

```diff
@@ -941,7 +941,11 @@ func (k *KubernetesUtils) Cleanup(namespaces []string) {
 
 	for _, ns := range namespaces {
 		log.Infof("Deleting test Namespace %s", ns)
-		if err := k.clientset.CoreV1().Namespaces().Delete(context.TODO(), ns, metav1.DeleteOptions{}); err != nil {
+		// Using Poll to ensure namespace is deleted successfully, because sometimes
+		// it takes more time to delete a namespace and if that action is overridden by
+		// some new task then the contents of the namespace are not deleted properly and,
+		// a race condition may arise for further tests.
```

Comment:
The comment is not all correct but I don't think it needs to explain so many details for a delete operation. Deleting a resource and waiting for it to complete is straightforward to understand. But as suggested in https://github.com/antrea-io/antrea/pull/3417#issuecomment-1063568983, could you add a stats to `deleteNamespace` so we can know how long it takes to delete a namespace? Something like:
```
	startTime := time.Now()
	defer func() {
		log.Infof("Deleting Namespace %s took %v", ns, time.Since(startTime))
	}()
```

Reply:
yes sure